src/js/pages/MyProfile/components/Trades.jsx

```diff
@@ -4,9 +4,31 @@ import {Card, Row, Col} from 'reactstrap';
 import {Link} from "react-router-dom";
 import {withNamespaces} from 'react-i18next';
 import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
-import {faCircle, faArrowRight} from "@fortawesome/free-solid-svg-icons";
+import {faArrowRight} from "@fortawesome/free-solid-svg-icons";
 import Identicon from "../../../components/UserInformation/Identicon";
 import {formatBalance} from "../../../utils/numbers";
+import {tradeStates} from "../../../features/escrow/helpers";
+
+const getTradeStyle = (tradeState) => {
+  switch(tradeState){
+    case tradeStates.waiting:
+    case tradeStates.funded:
+    case tradeStates.paid:
+      return {text: 'Open', color: '#44D058'};
+    case tradeStates.released:
+      return {text: 'Closed', color: '#4360DF'};
+    case tradeStates.canceled:
+      return {text: 'Canceled', color: '#939BA1'};
+    case tradeStates.expired:
+      return {text: 'Expired', color: '#939BA1'};
+    case tradeStates.arbitration_open:
+      return {text: 'Arbitration', color: '#FF2D55'};
+    case tradeStates.arbitration_closed:
+      return {text: 'Resolved', color: '#4360DF'};
+    default:
+      return {text: tradeState, color: '#939BA1'};
```

Comment:
Would be better to use a class name instead of putting the color codes here. You can see in variable-overrides.scss, all the colors are already there: https://github.com/status-im/status-teller-network/blob/master/src/css/variable-overrides.scss#L39-L49

Most of them already have classes too:
- blue: `text-primary`
- red : `text-danger`
- green: `text-success`

It's in the same file: https://github.com/status-im/status-teller-network/blob/master/src/css/variable-overrides.scss#L72-L80